components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/util/IdPManagementConstants.java

```diff
@@ -604,6 +604,14 @@ public static class SQLQueries {
                 "SELECT * FROM IDP_AUTHENTICATOR WHERE TENANT_ID = ? AND DEFINED_BY = 'USER'";
     }
 
+    public static class WarningMessage {
+
+        public static final String WARN_STALE_IDP_ACTION = "The %s Identity Provider was deleted, but an error " +
```

Comment:
Let's rather say, Identity Provider: %s is deleted. but ....
Remove "The" in the plural form as well and use present tense